haunted-houses/src/main/java/cz/muni/fi/pa165/hauntedhouses/model/Specter.java

```diff
@@ -1,4 +1,132 @@
 package cz.muni.fi.pa165.hauntedhouses.model;
 
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+import java.util.Objects;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToMany;
+import javax.persistence.ManyToOne;
+import javax.persistence.Temporal;
+import javax.persistence.TemporalType;
+
+/**
+ * @author Zoltan Fridrich
+ */
+@Entity
 public class Specter {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
```

Comment:
Tady používáme GenerationType.SEQUENCE, na cvičení nám říkali, že je to nejefektivnější způsob. Ale to je jenom detail.